akka-persistence/src/main/resources/reference.conf

```diff
@@ -161,3 +161,21 @@ akka.persistence.journal.leveldb-shared {
         native = on
     }
 }
+
+# LevelDB persistence requires the following dependency declarations:
+#
+# SBT:
+#       "org.iq80.leveldb"            % "leveldb"          % "0.7"
+#       "org.fusesource.leveldbjni"   % "leveldbjni-all"   % "1.7"
+#
+# Maven:
+#        <dependency>
+#            <groupId>org.iq80.leveldb</groupId>
+#            <artifactId>leveldb</artifactId>
+#            <version>0.7</version>
+#        </dependency>
+#        <dependency>
+#            <groupId>org.fusesource.leveldbjni</groupId>
+#            <artifactId>leveldbjni-all</artifactId>
+#            <version>1.7</version>
+#        </dependency>
```

Comment:
This should be included in the reference documentation, also.
It's a breaking change so it should also be mentioned in the migration guide.